### DISCUSSION TITLE
Access Denied Error 500

### DISCUSSION THREAD

**Discussion by AboutTimeIT** [akeeba/panopticon#284] [Q&A (Community Support)] [answered]
When in the Panoptocon V1.3 I go to "My Profile" I get an Access Denied Error 500
vendor/akeeba/awf/src/Mvc/ DataController.php (line 98)
PHP 8.1.24
Database server version 10.4.27-MariaDB-cll-lve
I have already performed a complete new install with the same result.

**Comment by nikosdion** [comment, marked as answer]
This a known issue which has already been fixed for version 1.0.4.
